lib/config/testdata/v18.xml

```diff
@@ -0,0 +1,15 @@
+<configuration version="17">
```

Comment:
No you're right it should be 18. Fixed. Also another thing, the override test was flaky.